docs/APISocketOptions.md

```diff
@@ -841,31 +868,72 @@ missing packet still wasn't recovered, or wasn't conditionally dropped (see
+`SRTO_INPUTBW` option set to 0 (automatic). Otherwise your stream will choke 
+and break quickly at any rise in packet loss.
 
-	- *Overhead is intended to give you extra bandwidth for the case when a packet 
-	has taken part of the bandwidth, but then was lost and has to be retransmitted. 
-	Therefore the effective maximum bandwidth should be appropriately higher than 
-	your stream's bitrate so that there's some room for retransmission, but still 
-	limited so that the retransmitted packets don't cause the bandwidth usage to 
-	skyrocket when larger groups of packets are lost*
+***TODO: set-only; get should be supported.***
 
-	- *Don't configure it too low and avoid 0 in the case when you have the 
-	`SRTO_INPUTBW` option set to 0 (automatic). Otherwise your stream will choke 
-	and break quickly at any rise in packet loss.*
 
-- ***To do: set-only; get should be supported.***
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_PACINGMODE
+
+| OptName               | Since | Binding | Type   | Units  | Default  | Range   | Dir | Entity |
+| --------------------- | ----- | ------- | ------ | ------ | -------- | ------- | --- | ------ |
+| `SRTO_PACINGMODE `    | 1.5.x | post    | `int`  |        |          | enum    | RW  | GSD    |
+
+Pacing control mode. This option is only applicable for live transmission mode.
+
+`SRTO_PACINGMODE` option defines the way of configuring the maximum allowed bandwidth `MAXBW`
+which limits the bandwidth usage by SRT and is used by live congestion control
+(LiveCC) module to calculate the minimum inter-sending time of consecutive packets.
+
+A combination of pacing control and live congestion control (LiveCC) modules controls SRT
+packets pacing based on the input rate and an overhead for packets retransmission
+in order to avoid congestion during fluctuations of the source bitrate.
+
+The following table shows the summary of pacing control modes, options need to be
+set (✓) or ignored (-) appropriately and formula for MAXBW calculation for a
+particular mode:
+
+| Mode / Option            | SRTO_MAXBW | SRTO_INPUTBW | SRT_OHEADBW | Formula for `MAXBW` Calculation                              |
+| ------------------------ | ---------- | ------------ | ----------- | ------------------------------------------------------------ |
+| `SRT_PACING_DEFMAXBW`    | -1         | -            | -           | `MAXBW` is set to the default value of 1Gbps                 |
+| `SRT_PACING_MAXBW`       | ✓          | -            | -           | `MAXBW` is set explicitly, in bytes per second               |
+| `SRT_PACING_INPUTBW`     | 0          | ✓            | ✓           | `MAXBW = INPUTBW * (1 + OHEADBW /100)`                       |
+| `SRT_PACING_ESTINPUTBW`  | 0          | 0            | ✓           | `MAXBW = ESTINPUTBW * (1 + OHEADBW /100)`                    |
+| `SRT_PACING_HYBRIDBW`    | 0          | ✓            | ✓           | `MAX_BW = max(INPUTBW, ESTINPUT_BW) * (1 + OHEADBW /100)`    |
```

Comment:
For convenience the other possible mode value `SRT_PACING_UNSET` should also be listed in the table.